commands/audit/jas/common_test.go

```diff
@@ -129,3 +129,24 @@ func TestSetAnalyticsMetricsDataForAnalyzerManager(t *testing.T) {
 		})
 	}
 }
+
+func TestCreateScannerTempDirectory(t *testing.T) {
+	scanner, cleanUp := InitJasTest(t)
+	defer cleanUp()
```

Comment:
```suggestion
	scanner := &JasScanner{TempDir: 'path'}
```
This is unit tests not integration. use stab